### PR TITLE
Spanish changes from translation team

### LINKED ISSUE #930: Task: Incorporate quality checks to already completed Spanish translation of header, footer, About, and Data & methodology
## Describe the task
Translation team provided document with a handful of subsequent translation corrections should be made after they ran a quality/context check on the site  [https://app.zenhub.com/files/358242767/f07826ea-d76c-4d55-beba-557bdae810a7/download](https://app.zenhub.com/files/358242767/f07826ea-d76c-4d55-beba-557bdae810a7/download)

## Acceptance Criteria

- [ ] first
- [ ] second
- [ ] third

## Additional context

* Add any other context about the task here.
* Or here

### PR DESCRIPTION
**Pull request by vim-usds**
# Purpose
- closes #930 

### QA Spec
- [x] On the About page: “Email” can be rendered in Spanish as “email” or as “correo electrónico”. For consistency with the term used in our translation, we recommend using “correo electrónico”, so this section reads “Correo electrónico: screeningtool.feedback@usds.gov”
- Links that are routed to English sites (6 occurrences) should have a prefix of [en ingles]

- [x] Footer link - FOIA
- [x] Footer link - RFI link
- [x] About page - paragraph 1 - Executive order link 1 
- [x] About page - paragraph 2 - Executive order link 2
- [x] About page - paragraph 3 - FAQ link
- [x] Link below the map

- [x] Percentile suffix of `th` in 65th percentile is replaced with `.a` in all occurrences. This is what the translation team mentioned:
```
For the census blocks: All other information in this section is translated into Spanish, 
but the ordinal numbers (highlighted in the screenshot) throughout the tool are 
abbreviated in English (e.g., 19th, 65th). We suggest changing this to the Spanish 
ordinal number abbreviation, which is .o for masculine and .a for feminine gendered 
words. ***Since this ranking applies to the communities of focus, which use a feminine 
gender in Spanish, we recommend that the th ordinal abbreviation in English be 
substituted with the feminine ordinal abbreviation in Spanish: .a  throughout the text. 
E.g., 19th would be 19.a in Spanish and 65th would be 65.a
```
I'm interpreting this to mean that all percentiles will have a `.a` suffix since they are referring to a feminine topic (communities of focus). Not sure if that is different now that it's `disadvantaged`? If so, we can easily change it to `.o`.  Interesting to learn that the suffix is not based on the number, but rather what the number represents. 🤔 
- [x] When a tract is selected, the methodology version on the top of the side panel is in Spanish.

### CODE REVIEW

File: client/src/components/Indicator/Indicator.tsx [hunk 1]
```diff
@@ -105,13 +103,34 @@ export const IndicatorValueSubText = ({value, isAboveThresh, threshold, isPercen
...
+ * In Spanish, the suffix is a ".a" and ".o", where only the "a" and "o" are superscripted.
+ * This function handles this case.
+ *
+ * Verbatim from translation team:
+ * We suggest changing this to the Spanish ordinal number abbreviation, which is .o for masculine
+ * and .a for feminine gendered words. ***Since this ranking applies to the communities of focus,
+ * which use a feminine gender in Spanish, we recommend that the th ordinal abbreviation in English
+ * be substituted with the feminine ordinal abbreviation in Spanish: .a  throughout the text.
+ * E.g., 19th would be 19.a in Spanish and 65th would be 65.a
+ *
```
Comment (esfoobar-usds) [inline client/src/components/Indicator/Indicator.tsx:118]: Amazing!

File: client/src/intl/es.json [hunk 2]
```diff
@@ -13,14 +13,15 @@
...
+    "about.page.sub.header.1.text.1": "En la <link1>Orden ejecutiva 14008 [en inglés]</link1> sobre <italictag>Hacer frente a la crisis climática en el país y en el extranjero</italictag>, el presidente Biden impartió instrucciones al Consejo sobre la Calidad del Medio Ambiente (CEQ, por sus siglas en inglés) de crear una Herramienta para la evaluación de la justicia climática y económica. El propósito de la herramienta es ayudar a las dependencias del gobierno federal a identificar comunidades desfavorecidas que se encuentran marginadas, desatendidas y abrumadas por la contaminación. La versión actual de la herramienta proporciona datos socioeconómicos, ambientales y climáticos para informar las decisiones que podrían afectar estas comunidades. La herramienta identifica a comunidades desfavorecidas a través de conjuntos de datos nacionales coherentes a disposición del público.",
```
Comment (esfoobar-usds) [inline client/src/intl/es.json:18]: Why is this in brackets? Should be parenthesis?

File: client/src/intl/es.json [hunk 3]
```diff
@@ -13,14 +13,15 @@
...
-    "about.page.sub.header.2.text.2": "Lea más sobre la Iniciativa Justice40 en el documento del presidente Biden en la <link1>Orden ejecutiva 14008</link1> sobre <italictag>Hacer frente a la crisis climática en el país y en el extranjero</italictag>.",
+    "about.page.sub.header.2.text.2": "Lea más sobre la Iniciativa Justice40 en el documento del presidente Biden en la <link1>Orden ejecutiva 14008 [en inglés]</link1> sobre <italictag>Hacer frente a la crisis climática en el país y en el extranjero</italictag>.",
     "about.page.sub.header.3.text": "¿Todavía tiene preguntas?",
-    "about.page.sub.header.3.text.1": "Encuentre las respuestas en la sección de <link1>Preguntas frecuentes</link1> de la Herramienta para la evaluación de la justicia climática y económica.",
+    "about.page.sub.header.3.text.1": "Encuentre las respuestas en la sección de <link1>Preguntas frecuentes[en inglés]</link1> de la Herramienta para la evaluación de la justicia climática y económica.",
```
Comment (esfoobar-usds) [inline client/src/intl/es.json:24]: Shoulb be parenthesis and missing a space between `frecuentes` and the symbol.

File: client/src/intl/es.json [hunk 4]
```diff
@@ -36,15 +37,15 @@
     "common.pages.footer.eng.cal.text": "Calendario de participación",
     "common.pages.footer.findcontact": "Buscar un contacto en USA.gov.",
     "common.pages.footer.findcontact.link": "https://www.usa.gov/espanol/",
-    "common.pages.footer.foia.text": "Ley de Libertad de Información (FOIA)",
+    "common.pages.footer.foia.text": "Ley de Libertad de Información (FOIA) [en inglés]",
```
Comment (esfoobar-usds) [inline client/src/intl/es.json:40]: Shouldn't be brackets.

File: client/src/intl/es.json [hunk 5]
```diff
@@ -36,15 +37,15 @@
...
     "common.pages.footer.gatsby.link": "https://github.com/usds/justice40-tool/blob/main/README-es.md",
     "common.pages.footer.github.link.text": "Vea el código en GitHub",
     "common.pages.footer.logo.title": "Consejo sobre la Calidad del Medio Ambiente",
     "common.pages.footer.moreinfoheader": "Más información",
     "common.pages.footer.privacy.link": "https://www.whitehouse.gov/es/privacidad/",
     "common.pages.footer.privacy.text": "Política de privacidad",
     "common.pages.footer.rfi.link": "https://www.federalregister.gov/d/2022-03920",
-    "common.pages.footer.rfi.text": "Solicitud de información",
+    "common.pages.footer.rfi.text": "Solicitud de información [en inglés]",
```
Comment (esfoobar-usds) [inline client/src/intl/es.json:48]: Shouldn't be brackets.

File: client/src/intl/es.json [hunk 6]
```diff
@@ -198,9 +199,9 @@
...
     "explore.map.page.side.panel.send.feedback.title": "Enviar comentarios",
     "explore.map.page.side.panel.spacer.and": "Y",
-    "explore.map.page.side.panel.version.title": "Versión de la metodología {versión}",
+    "explore.map.page.side.panel.version.title": "Versión de la metodología {version}",
     "explore.map.page.title.text": "Explore la herramienta",
-    "explore.map.page.under.map.download.draft.ptag.1": "<link1>Descargue la lista vigente</link1> de las comunidades y los conjuntos de datos que se usaron (el archivo .zip incluirá un archivo .xlsx y un archivo .csv, con un tamaño no comprimido de {downloadFileSize}). Última actualización: {dateUpdated}.",
+    "explore.map.page.under.map.download.draft.ptag.1": "<link1>Descargue la lista vigente [en inglés]</link1> de las comunidades y los conjuntos de datos que se usaron (el archivo .zip incluirá un archivo .xlsx y un archivo .csv, con un tamaño no comprimido de {downloadFileSize}). Última actualización: {dateUpdated}.",
```
Comment (esfoobar-usds) [inline client/src/intl/es.json:204]: Shouldn't be brackets.

**Review by esfoobar-usds — APPROVED** [review]
I added a question about brackets. If this is hypertext, then disregard all those bracket comments.

**Comment by vim-usds** [comment]
@KameronKerger and/or @katherinedm-usds 

Should we use square brackets vs parenthesis when designating a link will go to an English page (see footer [here](http://usds-geoplatform-justice40-website.s3-website-us-east-1.amazonaws.com/justice40-tool/1650-23894c/es/#3/33.47/-97.5)). The translation team recommended square brackets and Jorge recommended parenthesis. I am requesting for design and product to weight in. In some cases we already have something in parenthesis, in the FOIA link in the footer. How does consecutive parenthesis look?

**Comment by katherinedm-usds** [comment]
@vim-usds I'm doing QA now, this link seems to go to https://www.federalregister.gov/documents/2021/02/01/2021-02177/tackling-the-climate-crisis-at-home-and-abroad#:~:text=Sec.%20223.%20Justice40,40-percent%20goal. instead of EO
image

**Comment by vim-usds** [comment]
> @vim-usds I'm doing QA now, this link seems to go to https://www.federalregister.gov/documents/2021/02/01/2021-02177/tackling-the-climate-crisis-at-home-and-abroad#:~:text=Sec.%20223.%20Justice40,40-percent%20goal. instead of EO image

thank you taking a look now!

**Comment by KameronKerger** [comment]
CEQ requested that the link go to the fedreg and not the EO on wh.gov @katherinedm-usds

**Comment by katherinedm-usds** [comment]
thanks @KameronKerger !

Looks all good @vim-usds !!

**Pull request merged**